SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -771,10 +777,12 @@ void FSpatialNetGUIDCache::NetworkRemapObjectRefPaths(FUnrealObjectRef& ObjectRe
 
 void FSpatialNetGUIDCache::UnregisterActorObjectRefOnly(const FUnrealObjectRef& ObjectRef)
 {
-	FNetworkGUID& NetGUID = UnrealObjectRefToNetGUID.FindChecked(ObjectRef);
-	// Remove ObjectRef first so the reference above isn't destroyed
-	NetGUIDToUnrealObjectRef.Remove(NetGUID);
-	UnrealObjectRefToNetGUID.Remove(ObjectRef);
+	if (FNetworkGUID* NetGUID = UnrealObjectRefToNetGUID.Find(ObjectRef))
```

Comment:
Seems we only use this for stably named actors currently, under what circumstances might the object ref not be registered?

Reply:
This change is meant to catch an edge case where an object claims it's fully stably named (by overriding `IsFullNameStableForNetworking`) but actually isn't. It's not really a valid use case but still better not to crash.